libmscore/style.cpp

```diff
@@ -269,9 +269,11 @@ static const StyleType styleTypes[] {
       { Sid::vibratoPosAbove,         "vibratoPosAbove",         QPointF(.0, -1) },
       { Sid::vibratoPosBelow,         "vibratoPosBelow",         QPointF(.0, 1) },
 
-      { Sid::harmonyFretDist,          "harmonyFretDist",        Spatium(1.0) },
-      { Sid::minHarmonyDistance,       "minHarmonyDistance",     Spatium(0.5) },
-      { Sid::maxHarmonyBarDistance,    "maxHarmonyBarDistance",  Spatium(3.0) },
+      { Sid::harmonyFretDist,          "harmonyFretDist",            Spatium(1.0) },
+      { Sid::minHarmonyDistance,       "minHarmonyDistance",         Spatium(0.5) },
+      { Sid::maxHarmonyBarDistance,    "maxHarmonyBarDistance",      Spatium(3.0) },
```

Comment:
I don't see why those alignment changes are needed?

Reply:
No reason at all, is just a leftover when I had another, longer, name. Will be reverted.

Reply:
Done.